bin/check-licenses.mjs

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+
+/**
+ * External dependencies
+ */
+import { spawnSync } from 'node:child_process';
+
+/**
+ * Internal dependencies
+ */
+import { checkDepsInTree } from '../packages/scripts/utils/license.js';
+
+const ignored = [
+	'@react-native-community/cli',
+	'@react-native-community/cli-platform-ios',
+	'@ampproject/remapping',
+	'human-signals',
+	'fb-watchman',
+	'walker',
+	'chrome-launcher',
+	'lighthouse-logger',
+	'chromium-edge-launcher',
```

Comment:
This list is moved from the ignored list that was in package.json checking prod dependencies. What were prod dependencies are now the `wpScript` workspace packages.

Reply:
Is is still needed when RN dependencies are no longer listed as production?

Reply:
👍 I updated the list to remove all entries that do not error.